packages/volto/theme/themes/pastanaga/extras/grid.less

```diff
@@ -394,16 +394,16 @@ body.has-toolbar.has-sidebar .drag.gridBlock.handle.wrapper {
   }
 }
 
-// Aspect ratio on all images
-.grid-image-wrapper img,
-.gridBlock .block.image img,
-.block-editor-gridBlock .block-editor-image img,
-.block-editor-image.contained img {
-  width: 100%;
-  aspect-ratio: @grid-images-aspect-ratio;
-  object-fit: cover;
-  object-position: @grid-images-object-position;
-}
+// // Aspect ratio on all images
```

Comment:
@dobri1408  your comment due to "lint fix" will break the aspect ratio of the grid images.
Performing a git blame took me to this pull request
https://github.com/plone/volto/commit/28c6b288088c9bbddae4cff620d04d3cff2bafda#diff-639cb9d9fcdaa6fb08078cd2075da3eb8b0522d67912a00a0038a941b7573b68

Revert your comment

Reply:
@ichim-david I did it for him. Could I have a last blessing from you?